RadixWallet/Clients/BootstrapClient/BootstrapClient+Live.swift

```diff
@@ -7,6 +9,11 @@ extension BootstrapClient: DependencyKey {
 			bootstrap: {
 				appsFlyerClient.start()
 				homeCardsClient.bootstrap()
+
+				ImageDecoderRegistry.shared.register { context in
+					let isSVG = context.urlResponse?.url?.isSVG ?? false
+					return isSVG ? ImageDecoders.Empty() : nil
+				}
```

Comment:
I would rather keep this `bootstrap` method independent of specific logic. Until now, it just delegates the `bootstrap`/`start` calls to the corresponding clients (and it is agnostic of what AppsFlyer and HomeCards need to do when the app starts).

I don't think it justifies creating an `ImageDecodingClient` just for this method, so maybe we can do the following?

```Swift
// On new file called Nuke+Extensions or ImageDecoderRegistry+Extensions
extension ImageDecoderRegistry {
	static func bootstrap() {
		ImageDecoderRegistry.shared.register { context in
			let isSVG = context.urlResponse?.url?.isSVG ?? false
			return isSVG ? ImageDecoders.Empty() : nil
		}
	}
}

// On BootstrapClient (which removes the import Nuke)
{
    appsFlyerClient.start()
    homeCardsClient.bootstrap()
    ImageDecoderRegistry.bootstrap()
}
```

Reply:
Good idea! However, `import Nuke` will still be needed in `BootstrapClient` since we use `ImageDecoderRegistry` there.